pageObjects/RegistrationPage.js

```diff
@@ -0,0 +1,54 @@
+import { test, expect } from "@playwright/test";
+
+export default class RegistrationPage {
+    constructor(page) {
+      this.page = page;
+      this.signupNameInput = '#signupName';
+      this.signupLastNameInput = '#signupLastName';
+      this.signupEmailInput = '#signupEmail';
+      this.signupPasswordInput = '#signupPassword';
+      this.signupRepeatPasswordInput = '#signupRepeatPassword';
+      this.registerButton = 'text=Register';
+    }
+  
+    async fillRegistrationForm(name, lastName, email, password) {
+      await this.page.fill(this.signupNameInput, name);
+      await this.page.fill(this.signupLastNameInput, lastName);
+      await this.page.fill(this.signupEmailInput, email);
+      await this.page.fill(this.signupPasswordInput, password);
+      await this.page.fill(this.signupRepeatPasswordInput, password);
```

Comment:
page.fill is not recommended
![image](https://github.com/ChrissyDev/aqa-pw/assets/53534240/82000f6c-7b9d-4916-942d-7b6bf1dd4769)